modules/SofaGraphComponent/APIVersion.cpp

```diff
@@ -67,13 +68,13 @@ void APIVersion::init()
 void APIVersion::checkInputData()
 {
     if(!d_level.isSet() && !name.isSet() ){
-        msg_warning() << "The level is not set. Using 17.06 as default value. " ;
+        msg_warning() << "The level is not set. Using " << SOFA_VERSION_STR << " as default value. " ;
         return ;
     }
     if( !d_level.isSet() && name.isSet() ){
         d_level.setValue(getName());
     }
-    std::vector<std::string> allowedVersion = { "17.06", "17.12", "18.06", "18.12" } ;
+    std::vector<std::string> allowedVersion = { "17.06", "17.12", "18.06", "18.12", SOFA_VERSION_STR } ;
     if( std::find( allowedVersion.begin(), allowedVersion.end(), d_level.getValue()) == allowedVersion.end() )
     {
         msg_warning() << "The provided level '"<< d_level.getValue() <<"' is now valid. " ;
```

Comment:
What is the purpose of this?

Reply:
I don't know... I don't know what allowed versions are supposed to be for, but It didn't make sense to me to have old versions up to 18.12 and then suddenly stop. ^^